bin/resty

```diff
@@ -289,6 +289,11 @@ my $lua_shared_dicts = '';
                 next;
             }
 
+            if ($arg =~ /^--dump-nginx-conf$/) {
```

Comment:
I think we should just use the `eq` operator here since it's a constant string comparison.